test-container/src/main/java/io/strimzi/StrimziKafkaContainer.java

```diff
@@ -94,13 +112,18 @@ protected void doStart() {
         super.doStart();
     }
 
+    public StrimziKafkaContainer withExternalZookeeper(String connectString) {
+        this.externalZookeeperConnect = connectString;
+        return self();
+    }
```

Comment:
Perhaps private constructors and public factory methods to separate the two flavours would be better that this mutable field needing to be set before starting the container?